playbooks/setup_jenkins_slave.yml

```diff
@@ -26,11 +37,17 @@
         ClientAliveInterval: 15
         Compression: "yes"
   tasks:
+    - meta: flush_handlers
+
     - name: Install apt packages
       apt:
         pkg: "{{ item }}"
         state: installed
         update_cache: yes
+      register: install_packages
+      until: install_packages | success
+      retries: 5
+      delay: 2
```

Comment:
How are retries related here? While this is no harm, I'm just confused about why this is being done - perhaps to accommodate the potential for the connection failing (see comment in task above)?

Reply:
there seem to be issues running apt the first time, some times. I guess this could be a connection issue with SSH spinning back up, but it could also be a caching issue in the MNAIO. That said, the retry seems to ensure we get past this spot.

Reply:
I think we saw this in a job, and it was added as a result:

```
TASK [Install apt packages] ****************************************************
failed: [omnarktllp-18-fa33-infra1-vm] (item=[u'git-core', u'default-jre-headless', u'python-dev', u'gcc', u'libffi-dev', u'libssl-dev']) => {"failed": true, "item": ["git-core", "default-jre-headless", "python-dev", "gcc", "libffi-dev", "libssl-dev"], "module_stderr": "Warning: Permanently added '[104.130.18.4]:2222' (RSA) to the list of known hosts.\r\n/usr/bin/python: can't open file '/root/.ansible/tmp/ansible-tmp-1507037495.84-117600361617508/apt.py': [Errno 2] No such file or directory\n", "module_stdout": "", "msg": "MODULE FAILURE", "rc": 0}
	to retry, use: --limit @/var/lib/jenkins/workspace/OnMetal-Multi-Node-AIO_r14.2.0-kilo-trusty-leapfrogupgrade-large-periodic@3/rpc-gating/playbooks/setup_jenkins_slave.retry
```